bindings/cpp/src/lib.rs

```diff
@@ -798,10 +800,11 @@ fn restore_wallet_save_to_securestorage(
         password: password.clone(),
     };
 
-    let keyvalue = format!("{}_{}", servicename, username);
-
+    let keyvalue = format!("{}_{}", servicename, username); 
+    // need to hex encode to be included into json , without 0x prefix
```

Comment:
why is hexencode needed? json is already textual?

Reply:
if result value is json,  json inside json
{"result":""{"value.. }, caused  " parsing error.

textual expression is possible, i'll change

Reply:
it's changed to textual address, checked with rust